### DISCUSSION TITLE
Prodigy and Skin Cancer Detection

### DISCUSSION THREAD

**Discussion by kranthiksk** [explosion/spaCy#11987] [Help: Other Questions]
Hello,
I am new to Prodigy. I would like to know if there is a way to use Prodigy for Skin Cancer detection with Deep Learning Models. Can you please provide any resources regarding it?
Thank you

**Comment by polm** [comment]
Thanks for using Prodigy. 

You can use Prodigy to annotate image data which I guess you could use to detect skin cancer lesions, for example. But that's an area of research and you should probably look for papers or other resources. If you're new to computer vision, I would start with a tutorial on the topic in general.

Also note this forum is for spaCy, an open-source NLP library made by the same people as Prodigy. Computer vision is outside the scope of spaCy, so it'd be better to ask for help at the [Prodigy forums](https://support.prodi.gy/).

**Reply by kranthiksk** [reply]
Thank you @polm for your reply. I will look into it what you have said.
